docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+
+##### Priority Tags
+You can also set the perceived priority of an issue via tags. They are:
+- **Critical**: should only be used in the event of server-breaking bugs. 
+- **High**: game-breaking bugs with no workaround.
+- **Medium**: game-breaking bugs that do have a workaround.
+- **Low**: more typical bugs with quests/items/NPCs, etc. Your 'standard' bug.
+- **Trivial**: bugs that have no real impact on gameplay, typically cosmetic.
+
+## Guidelines
+- **Use your judgement** We are there to exercise our best judgement, not just as photocopiers. We have access to tools and information that most players don't, so we should be able to see more deeply into a problem than they can and make decisions accordingly.
+- **Generalize the problem.** Try to generalise or broaden a problem. If one unusual kind of item, NPC or spell is not working correctly,  try checking other items of the same general type to see if they are broken as well. Ideally we want to capture the broadest possible range of errors that we can.
+- **Be sceptical.** Users can feel very strongly about these issues. You will quite often see claims such as "it has always been this way" or "I've played my class since vanilla and this is how it should be." These assertions, with no other evidence, should be treated with an appropriate level of scepticism. An experienced player will probably know things about their class that you don't, but that does not mean their statements should be accepted as fact without any other evidence.
+- **Combine similar problems** - if you find related problems while searching, consider linking the two with a note. (It can be as simple as "See also <ticket URL>." This will help whoever ends up fixing the issue by telling them of similar issues.) 
+- **Try to prioritize** - more severe bugs over lesser ones, frequent bugs over rare ones. Counter-intuitively, newer issues may need priority over older ones, on the grounds that if a bug has lasted a long time without being fixed, there's a goood chance it's not very important.
+- If you can't find any sources, at least say so instead of saying nothing.
+- **Be polite.** When someone submits a bug report, they are doing us a favour. They've taken the time to make a GitHub account and type up a bug report to try to help us. The least we can do is be professional and courteous in return. 
+
+## Sources
+This is a general (and by no means exhaustive) look at the sources we can use to try to understand if a bug is valid or not. They include:
+
+- [TBC Wowhead](http://tbc.wowhead.com/) or [Current Wowhead](https://www.wowhead.com/) - Wowhead is a crucial source of data, both because of the site's own data and the comments that users have made on it. The Wowhead comments are probably the largest single set of information on individual quests and items we have, which is why it's a pity that so much of it is utter rubbish. 
```

Comment:
Wowhead is not crucial, it is a bad source of data.

Reply:
Unless you run it through Web Archives

Reply:
Comments from the right exp can be good. But looking at lootrates, spawns etc is bad because people run their addon on private servers.

Reply:
I didnt read the full page before making the comment, lol. You kinda touch on it tho good.

Reply:
heh, my fault for burying the lede. I've rewritten it slightly to mention the flaws in WH immediately instead of a few sentences in, and added the note about private servers contaminating the data.